model_analyzer/perf_analyzer/perf_config.py

```diff
@@ -103,6 +107,17 @@ def to_cli_string(self):
 
         return ' '.join(args)
 
+    def get_config_model(self):
```

Comment:
This function hopefully will not be necessary

Reply:
Removed.